Tests/SonarScanner.MSBuild.PreProcessor.Test/ProcessedArgsTests.cs

```diff
@@ -361,12 +390,86 @@ public void ProcArgs_OperatingSystem_AutoDetection(PlatformOS platformOS, string
             operatingSystemProvider.OperatingSystem().Returns(_ => platformOS);
             var sut = CreateDefaultArgs(operatingSystemProvider: operatingSystemProvider);
             sut.OperatingSystem.Should().Be(expectedOperatingSystem);
+            sut.IsValid.Should().BeTrue();
+        }
+
+        [TestMethod]
+        public void ProcArgs_UserHome_ParameterProvided()
+        {
+            var directoryWrapper = Substitute.For<IDirectoryWrapper>();
+            directoryWrapper.Exists(@"C:\Users\user\.sonar").Returns(true);
+            var sut = CreateDefaultArgs(new ListPropertiesProvider([new Property(SonarProperties.UserHome, @"C:\Users\user\.sonar")]), directoryWrapper: directoryWrapper);
```

Comment:
The argument has the same name as the parameter.
```suggestion
            var sut = CreateDefaultArgs(new ListPropertiesProvider([new Property(SonarProperties.UserHome, @"C:\Users\user\.sonar")]), directoryWrapper);
```

Reply:
Doesn't compile:
![image](https://github.com/SonarSource/sonar-scanner-msbuild/assets/103252490/92ecbc34-ce0e-4c47-a439-e95787113037)

Reply:
Indeed, I've missed the fact that there are parameters with default value skipped.